modules/customize-certificates-add-service-serving.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-{context}"]
+= Add a service certificate
+
+To secure communication to your service, have the cluster generate a 
+signed serving certificate and key pair into a secret in your namespace. To 
+do this, set the `service.beta.openshift.io/serving-cert-secret-name` 
+annotation on your service with the value set to the name you want to use 
+for your secret.
+
+.Procedure
+
+* Annotate the service with `service.beta.openshift.io/serving-cert-secret-name`.
++
+----
+$ oc annotate service <service-name> service.beta.openshift.io/serving-cert-secret-name=<service-name>
+----
++
+For instance, to annotate the service `foo` the following command would be
```

Comment:
s/the following command would be used:/use the following command:

>! would v
Use only to refer to an unfulfilled condition or hypothesis and to express the future in the past... Do
not use as an auxiliary verb...

Reply:
Corrected.